wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageSectionController.swift

```diff
@@ -233,7 +233,7 @@ final class ConversationMessageSectionController: NSObject, ZMMessageObserver {
         if isBurstTimestampVisible(in: context) {
             add(description: BurstTimestampSenderMessageCellDescription(message: message, context: context))
         }
-        if isSenderVisible || isTimeStampVisible, let sender = message.senderUser {
+        if isSenderVisible, isTimeStampVisible, let sender = message.senderUser {
```

Comment:
This means we'll only add the cell description if the both the sender and the timestamp are visible.

Reply:
Yes